cmd/root.go

```diff
@@ -86,12 +87,12 @@ func runKrokCmd(cmd *cobra.Command, args []string) {
 		Timestamp().
 		Logger()
 
-	//if krokArgs.server.GoogleClientID == "" {
+	// if krokArgs.server.GoogleClientID == "" {
```

Comment:
Oh I guess we can remove this, right? Did we agree in the end not to require this, right?

Reply:
Hmm could do @Skarlso. I think we'd need some sort of anonymous mode or something first though?

Reply:
Maybe :D